x-pack/plugins/security/server/routes/views/account_management.ts

```diff
@@ -13,4 +13,12 @@ export function defineAccountManagementRoutes({ httpResources }: RouteDefinition
   httpResources.register({ path: '/security/account', validate: false }, (context, req, res) =>
     res.renderCoreApp()
   );
+  httpResources.register(
+    { path: '/security/account/api-keys', validate: false },
```

Comment:
I could have just as easily tested, but does this support wildcards or optional path parameters? We could condense this into a single registration if so, instead of declaring them all individually.

Reply:
It supports path segments but we'd need regular expressions to be able to define specific paths that should be matched:

```typescript
  /**
   * E.g. if the router is registered at `/elasticsearch` and the route path is
   * `/search`, the full path for the route is `/elasticsearch/search`.
   * Supports:
   *   - named path segments `path/{name}`.
   *   - optional path segments `path/{position?}`.
   *   - multi-segments `path/{coordinates*2}`.
   * Segments are accessible within a handler function as `params` property of {@link KibanaRequest} object.
   * To have read access to `params` you *must* specify validation schema with {@link RouteConfig.validate}.
   */
```